tests/test_cmdline.py

```diff
@@ -22,6 +30,19 @@ def test_commandline_explicit_interp(tmpdir):
         str(tmpdir.join('venv'))
     ])
 
+# On Windows, path with Chinese characters are not working without a Chinese code page
+@pytest.mark.skipif("get_code_page() not in (None, 936, 950)")
+def test_commandline_non_ascii_path(tmpdir):
+    target_path = str(tmpdir.join('venv中文'))
+    if sys.version_info[0] == 2:
```

Comment:
Some problems exist in multi-byte locales only. See the updated test case in 57a43e823a7e32c7d717f918f12abebf89ffeb2a.